app/pods/agendas/controller.js

```diff
@@ -20,12 +27,86 @@ export default class AgendasController extends Controller.extend(DefaultQueryPar
   @action
   closeNewSessionModal() {
     this.isCreatingNewSession = false;
+    this.newMeeting.deleteRecord();
+    this.newMeeting = this.store.createRecord('meeting');
   }
 
   @action
-  successfullyAdded() {
+  async successfullyAdded() {
+    const agenda = await this.createAgenda(this.newMeeting);
+
+    const closestMeeting = await fetchClosestMeetingAndAgendaId(this.newMeeting.plannedStart);
+    const broader = await this.newMeeting.kind?.get('broader');
+    const isAnnexMeeting = broader?.uri === CONSTANTS.MEETING_KINDS.ANNEX;
+    if (!isAnnexMeeting && closestMeeting) {
+      await this.createAgendaitemToApproveMinutes(
+        agenda,
+        this.newMeeting,
+        closestMeeting
+      );
+    }
+    await this.newsletterService.createNewsItemForMeeting(this.newMeeting);
+    this.newMeeting = this.store.createRecord('meeting');
+    // TODO: Should fix sessionNrBug
+    // Import from meeting-utils.js
+    // await assignNewSessionNumbers();
```

Comment:
Is this comment still applicable @ValenberghsSven (and if yes, what does it entail) or can it be removed?

Reply:
Not sure what that bug is or was.
It seems that the relevant code in the session-number-service would delete the triple and reinsert a decimal `adms:identifier` (`meeting.number` in frontend). Knowing what we know now, decimal is not a correct type since we should use integer. 
So the comment can go, but we might have to look into cleaning up the type differences (and maybe find the bug) in new ticket?